README.md

```diff
@@ -176,7 +176,13 @@ $ y: echo -e "$((x+10))\n$((x+20))"
 
 ### Variable options
 
-For lines starting with `$` you can add extra options using `---`.
+For lines starting with `$` you can add use`---` to parse parameters to `fzf`.
+* `--allow-extra` : handles `fzf` option `--print-query`. `enter` will prefer a selection,
```

Comment:
Please add [alpha] or [experimental] because I'm still thinking about this API. maybe itll be the default in the future